collect_app/src/main/java/org/odk/collect/android/forms/FormUtils.java

```diff
@@ -65,4 +87,86 @@ public static void addSessionRootTranslators(ReferenceManager referenceManager,
             referenceManager.addSessionRootTranslator(rootTranslator);
         }
     }
+
+    public static ContentValues getValuesFromForm(Form form, StoragePathProvider storagePathProvider) {
+        ContentValues values = new ContentValues();
+        values.put(_ID, form.getId());
+        values.put(FormsProviderAPI.FormsColumns.DISPLAY_NAME, form.getDisplayName());
+        values.put(FormsProviderAPI.FormsColumns.DESCRIPTION, form.getDescription());
+        values.put(FormsProviderAPI.FormsColumns.JR_FORM_ID, form.getJrFormId());
+        values.put(FormsProviderAPI.FormsColumns.JR_VERSION, form.getJrVersion());
+        values.put(FormsProviderAPI.FormsColumns.FORM_FILE_PATH, storagePathProvider.getRelativeFormPath(form.getFormFilePath()));
```

Comment:
It only stores relative paths in the DB and returns `Form` objects with absolute paths. However, you could save an object with an absolute or a relative path it'd still work because I think because of how `StoragePathProvider#getRelativeFormPath` works. 

It seems like we now deal in absolute paths everywhere in our code however (as everything is going through the repository) so we shouldn't really have to think about any longer outside of `DatabaseFormsRepository`.